core/src/test/java/org/elasticsearch/common/util/BigArraysTests.java

```diff
@@ -37,8 +37,8 @@
 
 public class BigArraysTests extends ESSingleNodeTestCase {
 
-    public static BigArrays randombigArrays() {
-        final PageCacheRecycler recycler = randomBoolean() ? null : ESSingleNodeTestCase.getInstanceFromNode(PageCacheRecycler.class);
+    private BigArrays randombigArrays() {
```

Comment:
I only copied it because it didn't seem something general purpose that I would want to make easy for some other test to do, and having a couple 2 line methods seemed better. However, I don't feel strongly at all, so if you think it is better in single node testcase, I can move it there.